generic/generic.html

```diff
@@ -0,0 +1,303 @@
+                alt=""
+              />
+              <h2 class="description-h2-1">liability</h2>
+              <p class="description-p-1">
+                General Liability, Errors & Omissions, Employment Practices
+              </p>
+            </div>
+            <div class="details-container-1">
+              <img
+                class="description-img-1"
+                src="../images/logos/libaility-logo.png"
+                alt=""
+              />
+              <h2 class="description-h2-1">liability</h2>
+              <p class="description-p-1">
+                General Liability, Errors & Omissions, Employment Practices
+              </p>
+            </div>
+            <div class="details-container-1">
+              <img
+                class="description-img-1"
+                src="../images/logos/libaility-logo.png"
+                alt=""
+              />
+              <h2 class="description-h2-1">liability</h2>
+              <p class="description-p-1">
+                General Liability, Errors & Omissions, Employment Practices
+              </p>
+            </div>
+            <div class="details-container-1">
+              <img
+                class="description-img-1"
+                src="../images/logos/libaility-logo.png"
+                alt=""
+              />
+              <h2 class="description-h2-1">liability</h2>
+              <p class="description-p-1">
+                General Liability, Errors & Omissions, Employment Practices
+              </p>
+            </div>
+            <div class="details-container-1">
+              <img
+                class="description-img-1"
+                src="../images/logos/libaility-logo.png"
+                alt=""
+              />
+              <h2 class="description-h2-1">liability</h2>
+              <p class="description-p-1">
+                General Liability, Errors & Omissions, Employment Practices
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="form-getQuote-flex-container blue">
+          <h1>get yout quote online</h1>
+          <img
+            src="../images/banners/index-page/blueCenterDivider.png"
+            alt=""
+          />
+          <p>
+            Suspendisse et quam vitae est feugiat semper disse egestas velit
+            consectetur fermentumunc tellus convallis.
+          </p>
+          <label for="primary-industry">primary industry:</label>
+          <input id="primary-industry" type="text" />
+          <label for="primary-industry">primary industry:</label>
+          <input id="primary-industry" type="text" />
+          <label for="primary-industry">primary industry:</label>
+          <input id="primary-industry" type="date" />
+          <button class="button">get quote</button>
+        </div>
+      </div>
+      <div class="designDivider-bottom"></div>
+    </div>
+
+    <div class="bottom-banner-getQuote">
+      <div class="designDivider-top blue"></div>

```

Comment:
exista o chestie in CSS `:before` si `:after` pe care le poti folosi pt astfel de lucruri .. containeru asta nu adauga nici un fel de valoare HTML-ului